.gitignore

```diff
@@ -15,3 +15,11 @@
 !.yarn/sdks
 !.yarn/versions
 .pnp.*
+
+# Artefacts from running the daemons
```

Comment:
```suggestion
# Arteficts from running the daemons
```

Reply:
❓ 

 "artefact" (British & Australian) and "artifact" (US) are valid forms, but I'm not so sure about "artefict" 😀  I'm happy to change to US spelling if you wish so!